src/progress/pct/compile.p

```diff
@@ -212,15 +212,33 @@ END PROCEDURE.
 PROCEDURE initModule:
   ASSIGN lIgnoredIncludes = (LENGTH(cignoredIncludes) > 0).
 
-  IF (callbackClass > "") AND NOT bAboveEq113 THEN
+  IF (callbackClass > "") EQ TRUE AND NOT bAboveEq113 THEN
```

Comment:
@DustinGrau-PSC What's that ?

Reply:
That should handle the case where callbackClass may somehow be the unknown value ? as well as an empty string. It ensures that the value is truly a string.

Reply:
I've tried with multiple values (`?`, empty string, ...) and the `eq true` is not necessary

Reply:
Removed. It was more of a suggestion/habit when I use the > "" syntax for string checks.